sites/docs/src/content/docs/tutorials/tests_and_test_data/nf-test_writing_tests.md

```diff
@@ -128,24 +136,44 @@ nextflow_process {
     tag "abricate"
     tag "abricate/summary"
 
+    setup {
+        run("ABRICATE_RUN") {
+            script "../../run/main.nf"
+            process {
+            """
+            input[0] = Channel.fromList([
+                            tuple([ id:'test1', single_end:false ], // meta map
+                                file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/genome/genome.fna.gz', checkIfExists: true)),
+                            tuple([ id:'test2', single_end:false ],
+                                file(params.modules_testdata_base_path + 'genomics/prokaryotes/haemophilus_influenzae/genome/genome.fna.gz', checkIfExists: true))
```

Comment:
Should we also align every file and map horizontally ?
This would mean extra lines, but this look better and easier to review ?
```suggestion
                            tuple(
                                [ id:'test2', single_end:false ],
                                file(params.modules_testdata_base_path + 'genomics/prokaryotes/haemophilus_influenzae/genome/genome.fna.gz', checkIfExists: true)
                            )
```